pkgs/development/python-modules/ehtim/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  stdenv, lib, buildPythonPackage, fetchFromGitHub,
+  numpy, scipy, astropy, matplotlib, ephem, h5py, pandas, requests, future
+}:
+
+buildPythonPackage rec {
+  pname = "ehtim";
+  version = "1.2.4";
+
+  src = fetchFromGitHub {
+    owner = "achael";
+    repo = "eht-imaging";
+    rev = "v${version}";
+    sha256 = "ZeacOUve1QHX4Ei61GHEMtbqyfOV7Cj9q7+sH/lQR+w=";
+  };
+  patches = [ ./silence.patch ];
```

Comment:
Are there any other reasons to patch this out apart from making it less noisy? If not, we should leave it as-is, patching should be used primarily for fixing upstream bugs or making sure stuff works on a target platform.

Reply:
The reason was that it is not common that python packages print a greeting when being imported. But it is probably something that I should rather discuss with upstream.

Reply:
I don't think that this is a compelling reason to patch this behavior out by default, I believe we should keep upstream's intentions in this regard. A user can always patch that locally if needed.